# remove referencesto win32 and osx code we don't build
# patch originally from fedora, adapted to new versions
--- org/gudy/azureus2/platform/PlatformManagerFactory.java.orig	2007-01-26 12:59:30.000000000 -0600
+++ org/gudy/azureus2/platform/PlatformManagerFactory.java	2007-01-26 13:00:08.000000000 -0600
@@ -40,46 +40,9 @@
 	public static PlatformManager
 	getPlatformManager()
 	{
-		try{
-			class_mon.enter();
-		
-			if ( platform_manager == null ){
-										  
-				try{
-					if ( getPlatformType() == PlatformManager.PT_WINDOWS ){
-						
-						platform_manager = org.gudy.azureus2.platform.win32.PlatformManagerImpl.getSingleton();
-						
-					}else if( getPlatformType() == PlatformManager.PT_MACOSX ){
-						
-	                    platform_manager = org.gudy.azureus2.platform.macosx.PlatformManagerImpl.getSingleton();
-	                    
-					}else if( getPlatformType() == PlatformManager.PT_UNIX ){
-						
-            platform_manager = org.gudy.azureus2.platform.unix.PlatformManagerImpl.getSingleton();
-
-					}
-				}catch( PlatformManagerException e ){
-					
-						// exception will already have been logged
-					
-				}catch( Throwable e ){
-					
-					Debug.printStackTrace(e);
-				}
-			}
+		platform_manager = org.gudy.azureus2.platform.dummy.PlatformManagerImpl.getSingleton();
 			
-			if ( platform_manager == null ){
-				
-				platform_manager = org.gudy.azureus2.platform.dummy.PlatformManagerImpl.getSingleton();
-			}
-			
-			return( platform_manager );
-			
-		}finally{
-			
-			class_mon.exit();
-		}
+		return( platform_manager );
 	}
 	
 	public static int
